modules/hitlnext/src/backend/middleware.ts

```diff
@@ -81,10 +91,12 @@ const registerMiddleware = async (bp: typeof sdk, state: StateType) => {
 
   const handleIncomingFromAgent = async (handoff: IHandoff, event: sdk.IO.IncomingEvent) => {
     const { botAvatarUrl }: Config = await bp.config.getModuleConfigForBot(MODULE_NAME, event.botId)
+    const { attributes } = getCachedAgent(handoff.agentId)
```

Comment:
what if we have a cache miss here because agent in cache has been bumped because of maxAge ? 

from the docs 
```
maxAge Maximum age in ms. Items are not pro-actively pruned out as they age, but if you try to get an item that is too old, it'll drop it and return undefined instead of giving it to you. Setting this to a negative value will make everything seem old! Setting it to a non-number will throw a TypeError.
```

We might not want to use an LRU cache for this and maybe more of a simple agent dic instead

Reply:
Good catch, going with a simple dic seems fine and solves the problem.